### ISSUE TITLE
how to load model?

### ISSUE THREAD

**Issue by dreamyou070** [ryanwongsa/Sign2GPT#13]
I want to use the model,
but there is no   "models.trial_models.test_pretraining" as https://github.com/ryanwongsa/Sign2GPT/blob/main/configs/standards/standard_meta_model_zero_config.py

If I want to load model, how should i do ?

Please check once!
Thanks!

**Comment by ryanwongsa** [comment]
I added an example of how to load the pseudo-gloss pretraining model in file: `test_pretraining_model.py` in PR #14 

Let me know if you needed anything else.

**Comment by dreamyou070** [comment]
Thanks for check the model,
However there is no data like

csv_dir = [f"data/phoenix2014t/PHOENIX-2014-T.{x}.corpus.csv" for x in ["train", "dev", "test"]]
csv_dir = [f"data/csldaily/data.tsv"]

(from scripts folder)

Thanks!

**Comment by dreamyou070** [comment]
> The phoenix csvs need to be obtained from the original PHOENIX2014T dataset. The csldaily `data.tsv` is created from `scripts/csldaily_tsv_processing.py` but to obtain the original pickle you need to request access from the original authors of the CSL-DAILY dataset.

i success!

**Comment by ryanwongsa** [comment]
> I tried to inference with the sample code.
> But it takes so much time only when loading model and I couldn't success on inference
> Can you check once ?

Could you explain which part you are trying to train, as the example model I gave was for the pseudo-gloss pretraining component?

As for your screenshot, it doesn't look like it shows an error yet, the initial model tries to download a few GB for fastText embeddings which might take a while depending on your internet speed.

**Comment by dreamyou070** [comment]
when inference, I used the same code as you published..

Also,,, Can you share the training code if possible?
I want to train with English mode!

**Comment by ryanwongsa** [comment]
I plan to release, the full training script some time by April. It is possible that you can use the models and loss in your own training script, as those parts have all been already uploaded.

As for the English model, the datasets I trained my model on where Phoenix2014T and CSL-Daily, which are German and Chinese. Therefore, you will probably need to create your own pipeline for an English dataset.

**Comment by ryanwongsa** [comment]
Since this is related to only how to load the model, I will close this issue. But I have also updated the code in the main branch.
